docs/source/deployorderer/ordererchecklist.md

```diff
@@ -0,0 +1,307 @@
+
+**`ListenPort`**
+**`ListenAddress`**
+**`ServerCertificate`**
+**`ServerPrivateKey`**
+
+## General.BoostrapMethod
+
+```
+# Bootstrap method: The method by which to obtain the bootstrap block
+# system channel is specified. The option can be one of:
+#   "file" - path to a file containing the genesis block or config block of system channel
+#   "none" - allows an orderer to start without a system channel configuration
+BootstrapMethod: file
+```
+
+**`BootstrapMethod`**: (default value should not be overridden) Unless you plan to use a file type other than “file”, this value should be left as is.
+
+## General.BoostrapFile
+
+```
+# Bootstrap file: The file containing the bootstrap block to use when
+# initializing the orderer system channel and BootstrapMethod is set to
+# "file".  The bootstrap file can be the genesis block, and it can also be
+# a config block for late bootstrap of some consensus methods like Raft.
+# Generate a genesis block by updating $FABRIC_CFG_PATH/configtx.yaml and
+# using configtxgen command with "-outputBlock" option.
+# Defaults to file "genesisblock" (in $FABRIC_CFG_PATH directory) if not specified.
+BootstrapFile:
+```
+
+**`BoostrapFile`**: (default value should be overridden) Specify the location and name of the ordering service genesis block to use when this node is created.
+
+## General.LocalMSPDir
+
+```
+# LocalMSPDir is where to find the private crypto material needed by the
+# orderer. It is set relative here as a default for dev environments but
+# should be changed to the real location in production.
+LocalMSPDir: msp
+```
+
+**`LocalMSPDir`**: (default value should not be overridden) If you change this from `msp`, your folder structure must reflect the folder name you have specified here. The pathname is relative to the `FABRIC_CFG_PATH` or you can specify an absolute path.
+
+## General.LocalMSPID
+
+```
+# LocalMSPID is the identity to register the local MSP material with the MSP
+# manager. IMPORTANT: The local MSP ID of an orderer needs to match the MSP
+# ID of one of the organizations defined in the orderer system channel's
+# /Channel/Orderer configuration. The sample organization defined in the
+# sample configuration provided has an MSP ID of "SampleOrg".
+LocalMSPID: SampleOrg
+```
+
+**`LocalMSPID`**: (default value should be overridden) The MSP ID must match the orderer organization MSP ID that exists in the configuration of the system channel. This means the MSP ID must have been listen in the `configtx.yaml` used to create the genesis block of the system channel (or have been added later to the list of system channel administrators).
+
+## General.BCCSP.*
+
+BCCSP.SW can’t be configured at the moment. Leave those parameters as they are.
+Should this section be similar to the same content in the  CA and Peer sections?
+
+**`PKCS11`**:
```

Comment:
PKCS11 resides inside the BCCSP section no alongside it. I think we can make these recommendations match the peer recommendations for these parameters instead?